src/OpenTK.GraphicsLibraryFramework/Interfaces/IGLFW.cs

```diff
@@ -20,6 +22,7 @@ internal interface IGLFW
     {
         bool Init();
         void Terminate();
+        
         void InitHint(InitHint hint, int value);
```

Comment:
I don't know how you actually planned this. But this function(and some others) are only available in GLFW version 3.3 whereas https://github.com/opentk/opentk/pull/867/files#diff-927ba3ed0c05a5204ad8c549ee2d57f6R195 throws an exception because we do not support 3.3 yet.
So question is, was it concluded to use GLFW 3.3 even if it was not in release yet? Or do you just want to implement the API beforehand that it can be used when it is available?

Reply:
Just implemented those functions for completeness. We're not going to use any GLFW 3.3 features in OpenTK itself until 3.3 is stable.